samples_project/Assets/SampleViewer/Samples/Routing/RouteManager.cs

```diff
@@ -310,4 +311,18 @@ private void RebaseRoute()
         }
     }
 
+    public void ClearLineMarkers()
+    {
+        ClearRoute();
+
+        foreach (var routeMaker in routeMarkers)
+            Destroy(routeMaker);
```

Comment:
On line 264 brackets are added, here there are no brackets. Should be made consistent with the other change.